app/services/controller.py

```
@@ -31,9 +31,15 @@ def fetch_channel(id) -> dict:
     Returns:
         channel_details[dict]: [the details of the channel requested]
     """
+    # try:
     channel_details = dict(channel_DB().find_one({"_id": ObjectId(id)}))
-    channel_details.pop("_id")
-    return channel_details
+    if channel_details is None:
+        return "channel not found, wrong id"
```

raise `custom expections` and catch those instead of returning random strings like these. @ChochChill @sadhana1058